resources/scss/ceres/widgets/Checkout/_payment-and-shipping-list-widget.scss

```diff
@@ -0,0 +1,43 @@
+@mixin widget-list-box-shadow($color)
+{
+    > .payment-method-select, > .shipping-method-select
```

Comment:
In einem allgemeinen Mixin macht die Einschränkung auf '.payment-method-select' bzw. '.shipping-method-select' keinen Sinn. Ist diese Selektor-Ebene überhaupt notwendig? Falls ja, bitte auf allgemeingültige Klasse umbauen (z.B. '.provider-select'), sonst wenn möglich bitte entfernen.